src/main/java/frc/robot/OperatorInterface.java

```diff
@@ -1,115 +1,172 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
-/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* Copyright (c) 2021-2022 FIRST. All Rights Reserved. */
+/* Open Source Software - may be modified and shared by FRC teams. The code */
 /* must be accompanied by the FIRST BSD license file in the root directory of */
-/* the project.                                                               */
+/* the project. */
 /*----------------------------------------------------------------------------*/
 
 package frc.robot;
 
 import edu.wpi.first.wpilibj.Joystick;
 
 public class OperatorInterface {
-    public Joystick pilot, copilot;
-    private DTButton[] pilotButtons, copilotButtons, cocopilotButtons;
-    private static boolean isPilotDPadPressed = false;
-    private static boolean isCopilotDPadPressed = false;
+    private boolean isPilotDPadPressed;
+    private boolean isCopilotDPadPressed;
 
+    public final Joystick pilot;
+    public final Joystick copilot;
+
+    private final DTButton[] pilotButtons;
+    private final DTButton[] copilotButtons;
+
+    // optional
+    private DTButton[]    cocopilotButtons;
+    private final boolean cocopilotButtonsActive;
```

Comment:
We don't have a cocopilot so probably can drop these and the methods that use them.